host_guest/SAMPLing/absolute_submission_example.txt

```diff
@@ -1,12 +1,17 @@
-# SAMPLing challenge results
+# SAMPLing challenge results for absolute free energy calculations
 #
 # This file will be automatically parsed. It must contain the following five sections: predictions, computational cost,
 # software listing, technical details, and method description. These elements must be provided in the order shown.
 #
-# The data in each prediction line should be a comma-separated list of exactly 201 values, 100 of which are optional.
+# The data in each prediction line should be a comma-separated list of exactly 201 values, 100 of which may or may not be filled.
```

Comment:
I still think it would be clearer if you said "exactly 201 fields, 101 of which are optional" or similar. A field has a value or not, but I don't see how a value is optional.

Reply:
Ah! Sorry I didn't notice in your previous review you substituted "values" with "fields". I'll fix it before merging.